pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/fraser/Omsorgsopptjening.kt

```diff
@@ -0,0 +1,103 @@
+                English to "What are pension earnings for care work?"
+            )
+        }
+        paragraph {
+            val nettside = felles().select(Felles::avsenderEnhet).select(NAVEnhet::nettside)
+            textExpr(
+                Bokmal to "Pensjonsopptjeningen fra omsorgsarbeidet tilsvarer det du ville fått av en inntekt på 4,5 ganger grunnbeløpet for det aktuelle året. Du kan lese mer om dette på nav.no, og få oversikt over pensjonsopptjeningen din på ".expr() +
+                        nettside + ". Her finner du også omsorgsopptjening som du har fått godkjent.",
+                Nynorsk to "Pensjonsoppteninga frå omsorgsarbeidet svarer til det du ville ha fått av ei inntekt på 4,5 gonger grunnbeløpet for det aktuelle året. Du kan lese meir om dette på nav.no, og få oversikt over pensjonsoppteninga di på ".expr() +
+                        nettside + ". Her finn du også omsorgsopptening som du har fått godkjent.",
+                English to "The pension savings you earn for care work are equivalent to what you would have earned on an income of 4.5 times the National Insurance basic amount (“G”) for the year in question. You can read more about this at nav.no and see your earned pension savings at ".expr() +
+                        nettside + ". Here you will also see your approved pension earnings for care work. "
+            )
+        }
+    }
+
+    // omsorgsopptjenOverforingInfoOverskrift_001, omsorgsopptjenOverforingInfo_001
+    val overforingInfo = OutlinePhrase<LangBokmalNynorskEnglish, Unit> {
+        title1 {
+            text(
+                Bokmal to "Overføring av omsorgsopptjening til den andre forelderen",
+                Nynorsk to "Overføring av omsorgsopptening til den andre forelderen",
+                English to "Transferring pension earnings for care work to the other parent"
+            )
+        }
+        paragraph {
+            text(
+                Bokmal to "Den av foreldrene som får barnetrygden vil automatisk få omsorgsopptjeningen. Opptjeningen kan overføres til den andre forelderen hvis dere er sammen om omsorgen for barnet. Da må dere fylle ut skjemaet «Overføring av omsorgsopptjening» (NAV 03-16.10), som dere finner på ${Constants.SKJEMA_URL}.",
+                Nynorsk to "Den av foreldra som får barnetrygda, får automatisk omsorgsoppteninga. Oppteninga kan overførast til den andre forelderen dersom de er saman om omsorga for barnet. Då må de fylle ut skjemaet «Overføring av omsorgsopptening» (NAV 03-16.10), som de finn på ${Constants.SKJEMA_URL}.",
+                English to "The child benefit recipient will automatically be credited the pension earnings for care work.” The pension earnings can be transferred to the other parent if you share the care of the child. To transfer pension earnings, complete the form “Transfer of pension earnings for care work” (form no. NAV 03-16.10), which you will find at ${Constants.SKJEMA_URL}."
```

Comment:
Er det en quote for mye? earnings for care work.>>”<<

Reply:
Good catch